README.md

```diff
@@ -1187,6 +1187,166 @@ var enrollProfileResponse = await challengeResponse.Remediation.RemediationOptio
                                             .ProceedAsync(enrollProfileRequest);
 ```
 
+#### Self-service signup flow
+
+Following factors are enrolled during the signup: password, security question (disliked food) and e-mail. All the others factors are optional and skipped.
```

Comment:
I suggest using the same description style as other scenarios to be consistent, plus 
`> Note: Steps to identify the user might change based on your Org configuration.`